modules/objdetect/test/test_qrcode_encode.cpp

```diff
@@ -450,6 +450,34 @@ TEST(Objdetect_QRCode_Encode_Decode_Structured_Append, DISABLED_regression)
 
 #endif // UPDATE_QRCODE_TEST_DATA
 
+TEST(Objdetect_QRCode_Encode_Decode_Structured_Append, regression)
```

Comment:
Add the number of the issue that is being fixed:
TEST(Objdetect_QRCode_Encode_Decode_Structured_Append, regression_22205)